app/src/main/java/org/dhis2/usescases/searchTrackEntity/ui/SearchScreenConfigurator.kt

```diff
@@ -6,6 +6,8 @@ import androidx.constraintlayout.widget.ConstraintSet
 import org.dhis2.R
 import org.dhis2.bindings.display
 import org.dhis2.bindings.dp
+import org.dhis2.commons.animations.hide
+import org.dhis2.commons.animations.show
```

Comment:
do we need these imports?

Reply:
Yep, since these are extension functions on `View`